refill-style.yaml

```diff
@@ -3344,18 +3207,18 @@ layers:
         other_country_boundary_disputed_etc:
             filter: { kind: [disputed, indefinite, indeterminate, lease_limit, line_of_control, overlay_limit] }
             draw:
-                dashed:
+                lines:
+                    style: dashed
```

Comment:
`dash` can be assigned in the `draw` group and we can eliminate `dashed` style

Reply:
done